libs/styles/article-card.css

```
@@ -0,0 +1,131 @@
+/* article card styles, share between blocks */
+
+.article-card {
+  cursor: pointer;
+  box-sizing: border-box;
+  display: inline-block;
+  /* border: solid 1px var(--color-gray-200); */
+  border-radius: 4px;
+  width: 100%;
+  min-width: 268px;
+  max-width: 378px;
+  margin-bottom: 24px;
+}
+
+@media (min-width: 600px) {
+  .article-card {
+    min-width: unset;
+    width: calc(100% / 2 - 16px);
+    margin-right: 32px;
+    margin-bottom: 32px;
+  }
+  .article-card:nth-of-type(2n),
+  .article-card:last-of-type {
+    margin-right: 0;
+  }
+}
+
+@media (min-width: 1200px) {
+  .article-card {
+    width: calc(100% / 3 - 22px);
+  }
+  .article-card:nth-of-type(2n) {
+    margin-right: 32px;
+  }
+  .article-card:nth-of-type(3n),
```

Blank line between sections
```suggestion
  }
  
  .article-card:nth-of-type(2n),
  .article-card:last-of-type {
    margin-right: 0;
  }
}

@media (min-width: 1200px) {
  .article-card {
    width: calc(100% / 3 - 22px);
  }
  
  .article-card:nth-of-type(2n) {
    margin-right: 32px;
  }
  
  .article-card:nth-of-type(3n),
```